setup.py

```diff
@@ -1,19 +1,21 @@
 #!/usr/bin/env python
 
 import os
+import setuptools  # noqa
 
-import setuptools
 from distutils.core import setup
 
-execfile('grouper/version.py')
+from grouper.version import __version__
```

Comment:
You might be able to use something else. i've seen things like `pkg_resources.get_distribution` be suggested. I've always used execfile out of simplicity. In general though it's not recommended to import your package in your setup.py.